runtime/compiler/runtime/DataCache.cpp

```diff
@@ -293,6 +293,19 @@ TR_DataCache* TR_DataCacheManager::allocateNewDataCache(uint32_t minimumSize)
                dataCache->_status = 0;
                dataCache->_vmThread = NULL;
                dataCache->_allocationMark = dataCacheSeg->heapAlloc;
+
+#ifdef LINUX
```

Comment:
This code is protected by #ifdef Linux, but in other files in this PR I have not seen the same ifdef protection.

Reply:
You are right. I better remove it for clarity.